docs/scos/dev/tutorials-and-howtos/howtos/howto-set-up-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,194 @@
+2. Optional: Define store-specific configuration:
+  1. For one or more stores you've defined in database, define a separate store-specific configuration. For example, `config/Shared/config-default_docker_de.php` is the configuration file for the `DE` store in the docker environment.
+  2. To apply the defined store-specific configuration, adjust the related deploy file in the `environment` section.
+
+  In the following example, the `docker_de` environment name points to the `config/Shared/config-default_docker_de.php` store-specific configuration file. For more information about this deploy file parameter, see [environment](/docs/scos/dev/the-docker-sdk/{{site.version}}/deploy-file/deploy-file-reference-1.0.html#environment):
+
+  ```yaml
+  ....
+
+  environment: docker_de
+
+  image:
+    tag:
+  ```
+
+3. To import data for the stores you've added, adjust all the [import files and import configuration](/docs/scos/dev/data-import/{{site.version}}/data-importers-overview-and-implementation.html).
+
+For example, define the import source for the `EU` region and  `DE` store you've added:
+
+```php
+# Setup data import for region 
+
+ - data_entity: store
+   source: data/import/common/EU/store.csv
+
+# Catalog Setup data import
+...
+- data_entity: product-price
+  source: data/import/common/DE/product_price.csv
+...   
+```
+
+4. Configure installation recipes
+
+Add the new regions to the installation recipes in `config/install/*` as follows:
+
+```yaml
+...
+
+regions:
+    ...
+    - {REGION_NAME}
+...    
+```
+
+For example, to add the `EU` and `US` region, adjust an installation recipe as follows:
+
+```yaml
+...
+
+stores:
+    ...
+    - EU
+    - US
+...    
+```
+
+Also, make sure that commands that need to trigger for each store have stores: `true` added, as in the following example. This is especially important for validation cache generation in the `docker.yml` file, as this is responsible for generating the validation cache when your application is deployed on the Spryker PaaS:
+
+```yaml
+...
+        rest-request-validation-cache-warmup:
+            command: 'vendor/bin/console rest-api:build-request-validation-cache'
+            stores: true
+...
+```
+
+## Configure the deploy file
+
+According to the desired infrastructure setup, configure the deploy file for the dynamic multi-store setup. In the following example, we configure the [dynamic multi-store setup 1](#multi-store-setup-infrastructure-options): database, search engine, and key-value storage are shared:
+
+Deploy file configuration for the dynamic multi-store setup 1:
+
+```yaml
+# ....
+
+regions:
+    EU:
+        services:
+            mail:
+                sender:
+                    name: Spryker No-Reply
+                    email: no-reply@spryker.local
+            database:
+                database: eu-docker
+                username: spryker
+                password: secret
```

Comment:
I know that we expose this password in the repo, but would be nice if we can add generic example here instead of saving our default password in web archive forever :)